### PR TITLE
connect up new intake Tof sensor to rumble on driver controller

### PR DESCRIPTION
**Pull request by josieammer**

### CODE REVIEW

**Comment by josieammer** [comment]
tested and works on robot.

File: ted/src/main/java/frc/robot/control/Constants.java [hunk 1]
```diff
@@ -214,8 +214,6 @@ public final class Constants {
     public static final int currentPowerDistributionPanelCanId = 29;
     public static final ModuleType currentPowerDistributionPanelType = ModuleType.kRev;
     public static final double overcurrentRumbleTimeSeconds = 0.25;
-    // TODO remove these if not needed
-    public static final int EveryBotMotorPdpPortId = 8;
-    public static final double EveryBotMotorMaximuCurrentAmps = 45.5;
```
Comment (mmcadamsms) [inline ted/src/main/java/frc/robot/control/Constants.java:219]: looks like some really old code there ... good catch

**Review by mmcadamsms — APPROVED** [review]
agree with your changes ... lets get them in before testing tonight, hope being we can lock down a bit after the practice tonight

File: ted/src/main/java/frc/robot/commands/IntakeAndFeedNoteCommand.java [hunk 2]
```diff
@@ -56,12 +81,21 @@ public void initialize()
...
+    rumbleStarted = false;
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
+    // rumble once note is detected in the intake
+    if (doRumble && !rumbleStarted) {
+      if (intake.isNoteDetected()){
```
Comment (mmcadamsms) [inline ted/src/main/java/frc/robot/commands/IntakeAndFeedNoteCommand.java:93]: This TOF is oriented same direction as the other 'shooter TOF sensors' right?

**Pull request merged**